operator/config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: arosvc.azurecr.io/aro-operator
+  newTag: 93b0c67f-dirty
```

Comment:
Is this being generated? If so can we put placeholder?

Reply:
@mjudeikis yes this will change when you run generate on a different commit. Placeholder is not a solution, as it will be replaced with a value when you run generate. So it needs to be cleaned up. We can think of something later to address this later. One way to do it is to ensure this is not committed into repo in a similar way we verified integrity of bindata in v3.